be/src/test/java/com/jjikmuk/sikdorak/integration/user/user/UserSearchIntegrationTest.java

```diff
@@ -0,0 +1,57 @@
+package com.jjikmuk.sikdorak.integration.user.user;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.jjikmuk.sikdorak.integration.InitIntegrationTest;
+import com.jjikmuk.sikdorak.user.user.controller.response.UserSimpleProfileResponse;
+import com.jjikmuk.sikdorak.user.user.service.UserService;
+import java.util.List;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+
+@DisplayName("유저 검색 통합테스트")
+class UserSearchIntegrationTest extends InitIntegrationTest {
+
+    @Autowired
+    private UserService userService;
+
+    @Test
+    @DisplayName("유저의 닉네임으로 검색할 경우 유저 검색결과를 반환한다.")
+    void search_users_by_full_nickname() {
+        String searchNickname = testData.forky.getNickname();
+        List<UserSimpleProfileResponse> result = userService.searchUsersByNickname(searchNickname);
+
+        assertThat(result).isNotEmpty();
+        assertThat(result.get(0).nickname()).isEqualTo(testData.forky.getNickname());
```

Comment:
p2; given, when 사이 개행으로 구분

```suggestion
        String searchNickname = testData.forky.getNickname();
        
        List<UserSimpleProfileResponse> result = userService.searchUsersByNickname(searchNickname);

        assertThat(result).isNotEmpty();
        assertThat(result.get(0).nickname()).isEqualTo(testData.forky.getNickname());
```